File: 11022 A+B - 8/11022.c
#include <stdio.h>
int main() {
	int testCount;
	scanf("%d", &testCount);
	for (int i = 0; i < testCount; ++i) {
		int a, b;
		scanf("%d %d", &a, &b);
		printf("Case #%d: %d + %d = %d\n", i + 1, a, b, a + b);
	}

	return 0;
}

File: 11022 A+B - 8/11022.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split('\n');
let testCount = parseInt(input[0]);

for (let i = 0; i < testCount; ++i) {
    let inputLine = input[i + 1].split(' ');
    let a = parseInt(inputLine[0]);
    let b = parseInt(inputLine[1]);

    console.log(`Case #${i+1}: ${a} + ${b} = ${a+b}`);
}